nifi-frontend/src/main/frontend/apps/nifi-registry/src/app/app-routing.module.ts

```diff
@@ -28,13 +28,16 @@ const routes: Routes = [
         path: 'explorer',
         loadChildren: () => import('./pages/resources/feature/resources.module').then((m) => m.ResourcesModule)
     },
+    {
+        path: 'buckets',
+        loadChildren: () => import('./pages/buckets/feature/buckets.module').then((m) => m.BucketsModule)
+    },
```

Comment:
Referring to our previous discussion involving the support of legacy urls, do we want to support a redirect from the old buckets path at `administration/workflow` to the new `buckets` path?